src/components/CytoscapeGraph/Layout/GroupCompoundLayout.ts

```diff
@@ -214,8 +213,8 @@ export default class GroupCompoundLayout {
 
       // Add and position the children nodes according to the layout
       parents.each(parent => {
-        // (4.a) Add back the children and the edges
-        this.cy.add(parent.scratch(CHILDREN_KEY));
+        // (4.a) Add back the child nodes (with edges still attached)
+        parent.scratch(CHILDREN_KEY).move(parent);
```

Comment:
If using `remove` above, we can use `restore` directly.
```suggestion
        parent.scratch(CHILDREN_KEY).restore();
```

Reply:
I still don't see how remove/restore is better than move (i.e. unset/set parent) ?  Is it just your preference?